src/conflict.rs

```diff
@@ -982,7 +982,14 @@ impl<'i, I: Interner> fmt::Display for DisplayUnsat<'i, I> {
 
                 // The only possible conflict at the root level is a Locked conflict
                 match conflict {
-                    ConflictCause::Constrains(_) | ConflictCause::ForbidMultipleInstances => {
+                    &ConflictCause::Constrains(version_set_id) => {
+                        writeln!(
+                            f,
+                            "{indent}The constraint at the root level {version_set} could not be fulfilled",
```

Comment:
I think its obvious for the user that the constraint is at the root level. I would just remove that part.

Reply:
The message is now `constraint "foo" cannot be fulfilled`. I added a test with a insta snapshot.